package.json

```diff
@@ -95,16 +95,16 @@
     "fbemitter": "^2.1.1",
     "file-loader": "^0.11.0",
     "flux": "^3.1.2",
-    "fury": "=3.0.0-beta.6",
-    "fury-adapter-apib-parser": "=0.10.0",
+    "fury": "^3.0.0-beta.9",
```

Comment:
We should lock the versioning.

I'd suggest to copy phoenix-ui `package-lock.json` here and doing `npm install --save` so that the unused modules get removed. That way the attributes-kit modules should get locked to whatever versions we submitted to TPLTA.

Reply:
We discussed it, I tried this approach but it doesn't work. NPM for some reason is trying to install `flatmap-stream@0.1.1` which doesn't exist

Reply:
I'd advise changing `^3.0.0-beta.9` to be fixed to `3.0.0-beta.9`, as this is a beta release of a major version there may (and probably will) be breaking changes in beta.10 etc and we don't want these to get upgraded.

Reply:
Ok I know whats happening here `flatmap-stream@0.1.1` was a package which contained ugly backdoor and was unreleased from NPM (https://github.com/dominictarr/event-stream/issues/115). The problem is that it is subdependency of `semantic-release@6.3.6`

Reply:
I knew I heard that somewhere